Documentation/source/Changelog.md

```diff
@@ -1,12 +1,20 @@
 Changelog
 =========
 
+4.0.0-beta.2 (2018-06-19)
+------------------
+
+-   Fixed issue with analysis
+-   Fixed tests performance
+-   Fixed issue with QR code scanning
+-   Added customization guide
```

Comment:
I cannot agree with this change log. It's too vague. It doesn't have to explain in detail, but "issue with analysis" means nothing to people.
What was the issue with the analysis?
Fixed tests taking too long to complete
Fixed an issue where QR codes were analysed twice